oci/spec_opts_unix.go

```diff
@@ -346,6 +346,15 @@ func WithUIDGID(uid, gid uint32) SpecOpts {
 	}
 }
 
+// WithSGID allows supplementary GIDs to be set for the Process
+func WithSGID(sgids []uint32) SpecOpts {
```

Comment:
I'd leave this `WithUIDGIDSGIDs` and make `WithUIDGID` call it, but @crosbymichael can decide as it was from his comment.